tests/unit/test_reporting.py

```diff
@@ -150,7 +156,11 @@ def test_set_status_gitlab(
             flexmock(),
             BaseCommitStatus.success,
             "We made it!",
-            MSG_MORE_DETAILS.format(url="https://api.packit.dev/build/111/logs"),
+            MSG_TABLE_HEADER_WITH_DETAILS
+            + create_table_content(
```

Comment:
Careful here, if the `create_table_content` is broken, the tests will pass. Ideally you could cover this by unit test «just for» the `create_table_content`